src/index.html

```diff
@@ -1,12 +1,481 @@
+            $2,199
+          </div>
+        </div>
+
+        <a href="#" class="card__buy">Buy</a>
+
+      </div>
+
+      <div class="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="Card Logo"
+          class="card__logo"
+        >
+
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+
+        <div class="card__review">
+
+          <div class="stars stars--4">
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+          </div>
+
+          <div class="card__review_number-of-reviews">
+            Reviews: 5
+          </div>
+
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">
+            Price:
+          </div>
+          <div class="card__price-amount">
+            $2,199
+          </div>
+        </div>
+
+        <a href="#" class="card__buy">Buy</a>
+
+      </div>
+
+      <div class="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="Card Logo"
+          class="card__logo"
+        >
+
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+
+        <div class="card__review">
+
+          <div class="stars stars--4">
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+          </div>
+
+          <div class="card__review_number-of-reviews">
+            Reviews: 5
+          </div>
+
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">
+            Price:
+          </div>
+          <div class="card__price-amount">
+            $2,199
+          </div>
+        </div>
+
+        <a href="#" class="card__buy">Buy</a>
+
+      </div>
+
+      <div class="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="Card Logo"
+          class="card__logo"
+        >
+
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+
+        <div class="card__review">
+
+          <div class="stars stars--4">
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+          </div>
```

Comment:
```suggestion
        <div class="card__review">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```

Add empty line between multiple sibling BLOCKS of html, example:

```
<li>
  <a>first link</a>
</li>

<li>
  <a>second link</a>
</li>
```

and don't add between single tags, example:

```
<span>hello</span>
<span>hello1</span>
```